src/appkit.cs

```diff
@@ -18025,6 +18061,36 @@ interface NSTextFieldCell {
 		bool WantsNotificationForMarkedText { get; set; }
 	}
 
+	[BaseType (typeof(NSTextFieldCell))]
+	interface NSTokenFieldCell {
+		[Export ("initTextCell:")]
+		IntPtr Constructor (string aString);
```

Comment:
Since that `init*` does not accept a `nil` string then it seems unlikely that `init` (the default constructor) is a valid option. 

In such case (in doubt) it's better to add `[DisableDefaultCtor]` on the type. It will be easy to add (in the future) if that's a valid case but it won't be a breaking change (and obsolete game) if it turns out to be broken (e.g. assert in a new macOS version).